bootstrap/tasks/optional/coredns.yaml

```diff
@@ -0,0 +1,34 @@
+---
```

Comment:
Should we put this in addons dir instead? I know it's not a toggle-able component but it would keep things in one place and I am fine with that.

Reply:
Could do since there's only one.